code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -1,6 +1,7 @@
 /obj/structure/closet/cardboard
 	name = "large cardboard box"
 	desc = "Just a box..."
+	icon = 'icons/obj/boxes.dmi'
```

Comment:
I'd go with `cardboard_box.dmi` because there's regular boxes---and if those ever get split off (not unlikely), this could cause confusion, one day.